rosdoc2/main.py

```diff
@@ -37,6 +39,7 @@ def main(sysargs=None):
     parser = argparse.ArgumentParser(
         description=f'{COMMAND_NAME} builds documentation for ROS packages'
     )
+    parser.add_argument('--version', action='version', version=f'{COMMAND_NAME} {VERSION}')
```

Comment:
Also, we can add `-v` as an alternative to `--version` if we want to.

Reply:
Yes please add the -v option, since that pattern has already been established with `-h, --help`